packages/dart/noports_core/lib/src/sshnp/sshnp.dart

```diff
@@ -1,32 +1,44 @@
 import 'dart:async';
 
 import 'package:at_client/at_client.dart' hide StringBuffer;
+import 'package:at_utils/at_logger.dart';
+import 'package:noports_core/src/common/streaming_logging_handler.dart';
 import 'package:noports_core/sshnp_foundation.dart';
 
 abstract interface class SshnpRemoteProcess {
   Future<void> get done;
+
   Stream<List<int>> get stderr;
+
   StreamSink<List<int>> get stdin;
+
   Stream<List<int>> get stdout;
 }
 
 abstract interface class Sshnp {
+  static final StreamingLoggingHandler _slh =
+      StreamingLoggingHandler(AtSignLogger.defaultLoggingHandler);
```

Comment:
Make one of our StreamingLoggingHandlers, and wrap the current defaultLoggingHandler in it